test/fixtures/alternative4-root-let-value-embed-field-success.dhall

```diff
@@ -1 +1,7 @@
-let x = { key1 = let __embed = ./spago.dhall in __embed.dependencies # ["newtype"] } in x.key1
\ No newline at end of file
+let x =
+      let __embed = { key1 = ./spago.dhall }
+
+      in  __embed
+        with key1.dependencies = __embed.key1.dependencies # [ "newtype" ]
+
+in  x.key1
```

Comment:
Stylistically, this is not the most "efficient" update we could make. Rather, the following would be a better way to handle this:
```dhall
let x = 
  { key1 = 
    let __embed = ./spago.dhall in 
    in  __embed with __embed.dependencies # ["newtype"]
  }

in  x.key1
```

Reply:
This has been resolved via the `Embed` case when level is `SearchingForField`.